itou/job_applications/models.py

```diff
@@ -87,6 +87,15 @@ def pending(self):
             ]
         )
 
+    def created_in_past_hours(self, hours):
+        """
+        Returns objects created during the specified past hours.
+        Perofrms an inclusive range query on `created_at` datetime field.
+        """
+        end_dt = timezone.now()
+        start_dt = end_dt - timezone.timedelta(hours=hours)
+        return self.filter(created_at__gte=start_dt, created_at__lte=end_dt)
```

Comment:
pourquoi `created_at__lte=end_dt` ? On risque d'avoir des candidatures dans le futur? A mon avis cette clause risque juste de rajouter des bugs, par exemple s'il y a une heure de différence à cause d'une subtilité de timezone alors bam on ignore les candidatures de la toute dernière heure o_O

Reply:
ah oui bien vu, je vais fixer

Reply:
je me suis mal exprimé, en fait je proposais de carrément virer la clause `created_at__lte=end_dt` et pas juste de la passer en `lt`

Reply:
of course 👍